From 02f214f8aefbd3ac299d029bc4353553dfe9ac24 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:54 -0400
Subject: [redhat] kabi: Adding symbol complete_and_exit (kernel/exit.c)

Message-id: <6f21a78798968fc88a3d5efefb64c53b44b54f01.1410510773.git.poros@redhat.com>
Patchwork-id: 96026
O-Subject: [PATCH RHEL7.1 054/233] kabi: Adding symbol complete_and_exit (kernel/exit.c)
Bugzilla: 1062361
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Function is used to terminating a kernel thread.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/complete_and_exit
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/complete_and_exit
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/complete_and_exit

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/complete_and_exit b/redhat/kabi/kabi-module/kabi_ppc64/complete_and_exit
new file mode 100644
index 0000000..1413e33
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/complete_and_exit
@@ -0,0 +1,2 @@
+#1-
+0x68609857 complete_and_exit vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/complete_and_exit b/redhat/kabi/kabi-module/kabi_s390x/complete_and_exit
new file mode 100644
index 0000000..aeaf8af
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/complete_and_exit
@@ -0,0 +1,2 @@
+#1-
+0xfb91066b complete_and_exit vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/complete_and_exit b/redhat/kabi/kabi-module/kabi_x86_64/complete_and_exit
new file mode 100644
index 0000000..86c2e17
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/complete_and_exit
@@ -0,0 +1,2 @@
+#1-
+0x5aeb145f complete_and_exit vmlinux EXPORT_SYMBOL
-- 
1.7.1